camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -502,6 +502,10 @@ export default class Tasklist extends Mixins(TaskListMixin) {
       reviewerList.data.forEach((user: UserPayload) => {
         this.reviewerUsersList.push(UserListObject(user));
       });
+      
+      const usersList = JSON.parse(JSON.stringify(this.reviewerUsersList));
+      console.log(usersList);
+      this.userSelected= usersList.find(((i: any)=>i.code?.includes(this.task.assignee)));
```

Comment:
Try to use more informative variables . Instead of i in arg use user. Userlist is mapped right so each individual object will be user obj